tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -180,6 +181,11 @@ def test_nodereplacement_proactive(self):
         # Verify everything running fine
         log.info("Verifying All resources are Running and matches expected result")
         self.sanity_helpers.health_check(tries=90)
+
+        # Verify OSD encrypted
+        if config.ENV_DATA.get("encryption_at_rest"):
+            osd_encryption_verification()
```

Comment:
Looks good. 
I think we need to add it also at the end of the function `test_nodereplacement_proactive_with_io_running`.